pkg/square/builder.go

```diff
@@ -6,24 +6,22 @@ import (
 	"fmt"
 	"sort"
 
-	"github.com/celestiaorg/celestia-app/pkg/appconsts"
-	"github.com/celestiaorg/celestia-app/pkg/blob"
-	"github.com/celestiaorg/celestia-app/pkg/inclusion"
-	"github.com/celestiaorg/celestia-app/pkg/namespace"
-	"github.com/celestiaorg/celestia-app/pkg/shares"
-	"github.com/tendermint/tendermint/pkg/consts"
-	coretypes "github.com/tendermint/tendermint/proto/tendermint/types"
+	"github.com/celestiaorg/go-square/pkg/blob"
+	"github.com/celestiaorg/go-square/pkg/inclusion"
+	"github.com/celestiaorg/go-square/pkg/namespace"
+	"github.com/celestiaorg/go-square/pkg/shares"
+	"google.golang.org/protobuf/proto"
 )
 
 type Builder struct {
-	// maxCapacity is an upper bound on the total amount of shares that could fit in the biggest square
-	maxCapacity int
+	// maxSquareSize is an upper bound on the total amount of shares that could fit in the biggest square
+	maxSquareSize int
```

Comment:
[blocking] I think this comment is now incorrect. Perhaps something like:

```suggestion
	// maxSquareSize is the maximum number of rows (or columns) in the original data square
	maxSquareSize int
```